app/src/main/java/org/ole/planet/myplanet/base/BaseContainerFragment.kt

```diff
@@ -190,9 +190,11 @@ abstract class BaseContainerFragment : BaseResourceFragment() {
             FileProvider.getUriForFile(MainApplication.context, "${MainApplication.context.packageName}.fileprovider", it)
         }
 
-        val intent = Intent(Intent.ACTION_INSTALL_PACKAGE)
-        intent.data = uri
-        intent.flags = Intent.FLAG_GRANT_READ_URI_PERMISSION or Intent.FLAG_ACTIVITY_NEW_TASK
+        val intent = Intent(Intent.ACTION_VIEW).apply {
+            data = uri
+            type = "application/vnd.android.package-archive"
```

Comment:
this two lines can be refactored to `setDataAndType(uri, "application/vnd.android.package-archive")` to remove the warnings the have